src/main/java/com/databasepreservation/modules/siard/in/read/ZipReadStrategy.java

```diff
@@ -49,7 +67,11 @@
 
         @Override public void setup(SIARDArchiveContainer container) throws ModuleException {
```

Comment:
Remove setup(SIARDArchiveContainer) from the ReadStrategy, make a lazy loading/setup when such is required by the other methods. Be aware that the lazy loading must be threat-safe (synchronized lazy load method).